### ISSUE TITLE
Does it work for web?

### ISSUE THREAD

**Issue by Hit1328** [ArjanAswal/Stockfish#42]
I was not able to get the example running on web but it did work for android. Please clarify if it can run on web as the documentation only describes about android and ios

**Comment by daohoangson** [comment]
No, only Android & iOS are supported right now. We probably need to use something like https://github.com/lichess-org/stockfish.wasm to make it work on web.

**Comment by Hit1328** [comment]
> No, only Android & iOS are supported right now. We probably need to use something like https://github.com/lichess-org/stockfish.wasm to make it work on web.

Aight! Initially I was developing a web app using [stockfish.js](https://github.com/nmrugg/stockfish.js) that required only a js and a wasm file. But later I thought of using flutter as I wanted to port it to all platforms. I have no idea if it is possible to integrate js and wasm for web inside flutter. Would be great if you guys can add a web support for the package. 
ps: I am beginner and working on my first big project.

**Comment by dafanasev1986** [comment]
Hi! Are there any plans for web support?

**Comment by CrispStrobe** [comment]
you can easily just use stockfish.js. here is a simple fork which enables builds on macos and includes a chess app example which builds on macos and, leveraging stockfish.js, for web: https://github.com/CrispStrobe/Stockfish
